synchronizer/l1_rollup_info_consumer_test.go

```diff
@@ -0,0 +1,80 @@
+package synchronizer
+
+import (
+	"context"
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/0xPolygonHermez/zkevm-node/etherman"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+)
+
+type consumerTestData struct {
+	sut      *l1RollupInfoConsumer
+	syncMock *synchronizerProcessBlockRangeMock
+	ch       chan l1SyncMessage
+	ctx      context.Context
+}
+
+func Test_Given_Consumer_When_ReceiveAFullSyncAndChannelIsEmpty_Then_StopOk(t *testing.T) {
```

Comment:
I have adopted the format: `Test_Given_XXX_When_YYY_Then_ZZZ` because, from my point of view, it's more easy to read. 
But if you disagree I can rename to this format `TestGivenXXWhenYYYThenZZZ`

Reply:
We agree to decide later